From 3d319e3a09dbd366d2721844cf6fb5d91adc76c0 Mon Sep 17 00:00:00 2001
From: Kevin Mihelich <kevin@archlinuxarm.org>
Date: Sun, 23 Apr 2017 16:56:55 -0600
Subject: [PATCH 4/4] Don't pair touchpad and lid switch

Kills Xorg immediately upon any touchpad or keyboard entry on the Samsung Chromebook Plus.

Signed-off-by: Kevin Mihelich <kevin@archlinuxarm.org>
---
 src/evdev-mt-touchpad.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/evdev-mt-touchpad.c b/src/evdev-mt-touchpad.c
index a35688b..2243682 100644
--- a/src/evdev-mt-touchpad.c
+++ b/src/evdev-mt-touchpad.c
@@ -1638,7 +1638,7 @@ tp_interface_device_added(struct evdev_device *device,
 
 	tp_pair_trackpoint(device, added_device);
 	tp_dwt_pair_keyboard(device, added_device);
-	tp_pair_lid_switch(device, added_device);
+	/*tp_pair_lid_switch(device, added_device);*/
 
 	if (tp->sendevents.current_mode !=
 	    LIBINPUT_CONFIG_SEND_EVENTS_DISABLED_ON_EXTERNAL_MOUSE)
-- 
2.12.2

